server/src/gateway/providers/slide.action.service.ts

```diff
@@ -26,6 +30,9 @@ export class SlideActionService {
     const boardId = client.data.user.targetBoard.boardId;
     const slideNumber = data.slide ? data.slide.slideNumber : 1;
     const slide = await this.slidesService.deleteSlide(boardId, slideNumber);
+    for (const obj of slide.objects) {
```

Comment:
isnt it better to perform deleting logic in the stats service and call the method once implemented?